### ISSUE TITLE
Training loss is nan

### ISSUE THREAD

**Issue by neney1234** [penny4860/YOLO-detector#22]
I have a problem with training these because after some time training loss become nan

**Comment by penny4860** [comment]
It is difficult to grasp only the contents of your inquiry. The phenomenon that the loss value goes to nan in the training process mainly occurs when hyper parameter tuning is wrong.
